custom/conf/app.example.ini

```diff
@@ -1388,6 +1388,12 @@ PATH =
 ;; Deliver timeout in seconds
 ;DELIVER_TIMEOUT = 5
 ;;
+;; Webhook can only call allowed hosts for security reasons. Comma separated list, eg: external, 192.168.1.0/24, *.mydomain.com
+;; Built-in: loopback (for localhost), private (for LAN/intranet), external (for public hosts on internet), * (for all hosts)
+;; CIDR list: 1.2.3.0/8, 2001:db8::/32
+;; Wildcard hosts: *.mydomain.com, 192.168.100.*
+;ALLOWED_HOST_LIST = external
```

Comment:
I don't think it can be backported with such default as that will break pretty much every instance that has any integrations hosted in local network, for example Drone CI, it could probably be set in 1.15 with more sensible default `external, private` to be less breaking change

Reply:
If the purpose is non-breaking, why not set it tl `*` including localhost?

Reply:
Yes, probably `*` would be most correct

Reply:
Not that I can think of use case for calling localhost but there could be one

Reply:
OK, we keep the legacy behavior in 1.15 and introduce the breaking setting in 1.16, then all users can be happy.

Will update later.

Reply:
This backport PR has been updated. I think I should open a new PR to fix the document for 1.16, to reminder users that the default value differs across versions.